README.md

```diff
@@ -1,3 +1,4 @@
+# Most Excellent branch
```

Comment:
This can't be merged into master because of this change. This generates a merge conflict. One thing we could do to get around this to re-commit by removing the change to the README.md.

So click the edit icon to remove this change and see if that fixes it! :+1: